README.md

```diff
@@ -2,46 +2,33 @@
-  - This configures the functions host to use the development version of the worker
-  - You can also configure `languageWorkers:node:workerDirectory` as an environment variable.
-- To debug, add the App Setting in local.settings.json `languageWorkers:node:arguments = --inspect-brk`
-  - You can also configure `languageWorkers:node:arguments` as an environment variable.
-
-Make sure that `languageWorkers:node:workerDirectory` and `languageWorkers:node:arguments` are set correctly. When you start your functions host, you should see your custom path to workerDirectory and any arguments you passed to the node. If it was **not set** correctly, your output may look like the default output: `Starting language worker process:node  "%userprofile%\AppData\Roaming\npm\node_modules\azure-functions-core-tools\bin\workers\node\dist/src/nodejsWorker.js" --host 127.0.0.1 --port 5134 --workerId fd9b17c3-8ffb-49f7-a4e3-089a780e7a00 --requestId 14e27374-9395-42d4-a639-bd67e0e770a4 --grpcMaxMessageLength 134217728`
-
-Read more on local debugging [in our docs](https://docs.microsoft.com/azure/azure-functions/functions-reference-node#local-debugging).
-
-## Publishing
-
-`package.ps1` creates the nuget package for the worker.
-
-It builds and webpacks the generated node files into a bundle.
-We include several grpc native modules, for x86/x64 versions of windows, osx, linux
-
-The nuget package can be deployed from the appveyor job at: https://ci.appveyor.com/project/appsvc/azure-functions-nodejs-worker
+- [Create your first Node.js function](https://docs.microsoft.com/azure/azure-functions/create-first-function-vs-code-node)
+- [Node.js developer guide](https://docs.microsoft.com/azure/azure-functions/functions-reference-node)
+- [Language Extensibility Wiki](https://github.com/Azure/azure-webjobs-sdk-script/wiki/Language-Extensibility)
 
 ## Contributing
 
+- Clone the repository locally and open in VS Code
+- Run "Extensions: Show Recommended Extensions" from the [command palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette) and install all extensions listed under "Workspace Recommendations"
+- Run `npm install` and `npm run build`
+- Create or open a local function app to test with
+- In the local function app, add the following settings to your "local.settings.json" file or configure them directly as environment variables
+  - `languageWorkers__node__workerDirectory`: `<path to the root of this repository>`
+  - `languageWorkers__node__arguments`: `--inspect`
+    > NOTE: If you need to debug worker initialization, use `--inspect-brk` instead of `--inspect`. Just keep in mind you need to attach the debugger within 30 seconds or the host process will timeout.
+- Start the local function app (i.e. run `func start` or press <kbd>F5</kbd>)
+- Back in the worker repository, press <kbd>F5</kbd> and select the process for your running function app
+

```

Comment:
Should we mansion "npm test" to ensure all tests are passed?

Reply:
Also maybe we need to add mocha profile so a contributor can set a breakpoint in a test

Reply:
Done. We already have a launch profile people can use for debugging unit tests here: https://github.com/Azure/azure-functions-nodejs-worker/blob/v3.x/.vscode/launch.json#L10